tests/load-tests.php

```diff
@@ -121,14 +121,32 @@ function( $module_settings ) {
 	}
 
 	public function test_perflab_get_generator_content() {
-		// Assert that it returns the current version and active modules.
+		// Assert that it doesn't returns dummy modules.
 		$dummy_active_modules = array( 'images/a-module', 'object-cache/another-module' );
```

Comment:
That's not the point of this test, this comment shouldn't be changed and the test still should ensure that it covers the previous comment.

We need to make sure of course that this test works with the new changes, but in order to do that we need to replace the usage of dummy modules here with real modules (for which the new can-load logic will work).

For example like this:
```suggestion
		// Assert that it returns the current version and active modules.
		// For this test, set the active modules to all defaults but the last one.
		$dummy_active_modules = require plugin_dir_path( PERFLAB_MAIN_FILE ) . 'default-enabled-modules.php';
		array_pop( $dummy_active_modules );
```

We should also rename `$dummy_active_modules` simply to something like `$active_modules`.

Reply:
Changes for the tests were reverted, and changes related to the active module were applied on PR. Thanks!